cogs/displayinfo.py

```diff
@@ -1,5 +1,4 @@
 import discord
```

Comment:
Final Comment: It's starting to look real good. I think the next big task for this feature (after all the small changes and other things to do) is to work on midnight timing. A couple of things to help avoid possible points of failure. 

0. We need a timer at midnight. For the following list, assume previous day is the now passed day and the new day is the current day past midnight. Once this timer goes off... 

Immediate/Soon Fixes/Changes:
1. Everyone that is still in an activity needs to have `_update_now()` called and a new day needs to be made for them by updating their databases.
2. Old startTimes of each person who had any activity of the previous day need to be removed so that we can save space in the database. Each person should ever only have "startTimes" in the current present day. 
3. The `self.current_date` needs to be updated to avoid issues with any calls.

Long term future changes and things to consider:

4. Time zones will be difficult. We may need to look into restructuring or figuring out how people in different time zones can have their database set. If we keep ever server and everyone on the same time zone, we will have a problem with people accessing their info from other time zones seeing that it doesn't match up with their day of 12 AM to 11:59 PM. Yet it will be very difficult to run the bot/database based on everyone's specific time zones.